Cargo.toml

```diff
@@ -11,4 +11,9 @@ resolver="1"
 exclude = [
     "bot_ci_cd",
     "foundry",
-]
\ No newline at end of file
+]
+
+
+[profile.release]
+codegen-units = 1
+lto = true
```

Comment:
I think we'll keep this only for now. Regarding the heap allocators I need to experiment more